OPCB/initServer.sqf

```diff
@@ -57,41 +55,12 @@ ZeusGroup = createGroup sideLogic;
 } forEach [officer_jeff,tank_spawner,heli_jeff]; 
 
 globalWaterPos = [3067.06,16839.7,10.1122]; //universal for all maps, has to be changed manually 
-WorldCenter = [5840,5700,0];
 
 CityMarker = createMarker ["citymarker",  getpos officer_jeff];
 
-Cities = nearestLocations [[5840,5700,0], ["NameCity","NameCityCapital","NameVillage"], 8000];
+Cities = nearestLocations [getPos jeff, ["NameCity","NameCityCapital","NameVillage"], 8000];
```

Comment:
On larger maps where the base is in the corner, 8 km away from Jeff might not cover the whole map. This command might interest you. https://community.bistudio.com/wiki/worldSize